src/index.ts

```diff
@@ -50,6 +51,8 @@ export async function notarize({ appPath, ...otherOptions }: NotarizeOptions) {
     } as NotaryToolStartOptions);
   }
 
+  await checkSignatures({ appPath });
```

Comment:
Should this go before notarizing, rather than before stapling? Notarization will fail if the signatures are invalid.

Reply:
Yes great point